[CommonServerPython & demistomock]  isTimeSensitive (#34196)
demisto/content: Packs/Base/ReleaseNotes/1_33_53.md

#### Scripts

##### CommonServerPython

> Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called as auto-extract=inline.

Author: ```suggestion
Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called with auto-extract=inline.
```

> Added the *is_time_sensitive* method, this method will indicate whether the command reputation (auto-enrichment) is called with auto-extract=inline.

Reviewer: ```suggestion
Added the *is_time_sensitive* method, indicating whether the current code is a reputation command called with auto-extract=inline.
```
